SwiftCompilerSources/Sources/Optimizer/Utilities/Test.swift

```diff
@@ -17,7 +17,7 @@
 //       }
 // - In SwiftCompilerSources/Sources/SIL/Test.swift's registerSILTests function,
 //   register the new test:
-//       registerFunctionTest(myNewTest, implementation: { myNewTest.run($0, $1, $2) })
+//       registerFunctionTest(myNewTest)
```

Comment:
Could you update this line to say a new argument should be added to the new variadic function in `regionsterOptimizerTests`?  ...and I see I neglected to update the function name two lines up to registerOptimizerTests from registerSILTests.

Reply:
Fixed the latter: https://github.com/apple/swift/pull/69054 .